### ISSUE TITLE
Is Mighty Supports ASP.NET Core 6?

### ISSUE THREAD

**Issue by comfreakph** [MightyOrm/Mighty#45]

**Comment by mikebeaton** [comment]
Good question - I really need to get to this - there is a fair amount of work integrating recent changes from my local branch, and making a few other fixes.

**Comment by jonasnordlund** [comment]
Yeah, ASP .NET Core support with a sample in the docs to use it with Dependency Injection would be great!

**Comment by mikebeaton** [comment]
The project definitely supports .NET Core with dependency injection already, though I guess additional docs for that could be good.

It also needs an update to be natively compiled for recent versions of .NET, which is what this issue is about.
